Protos/generate.sh

```diff
@@ -201,6 +201,25 @@ function generate_service_messages_interop_tests {
   done
 }
 
+function generate_worker_service {
+  local protos=(
+    "$here/upstream/grpc/testing/payloads.proto"
+    "$here/upstream/grpc/testing/control.proto"
+    "$here/upstream/grpc/testing/messages.proto"
+    "$here/upstream/grpc/testing/stats.proto"
+    "$here/upstream/grpc/testing/benchmark_service.proto"
+    "$here/upstream/grpc/testing/worker_service.proto"
+  )
+  local output="$root/Sources/performance-worker/Generated"
+  
+  generate_message "$here/upstream/grpc/core/stats.proto" "$here/upstream" "$output" "Visibility=Internal" "FileNaming=PathToUnderscores"
+
+  for proto in "${protos[@]}"; do
+    generate_message "$proto" "$here/upstream" "$output" "Visibility=Internal" "FileNaming=PathToUnderscores"
+    generate_grpc "$proto" "$here/upstream" "$output" "Visibility=Internal" "Server=true" "_V2=true" "FileNaming=PathToUnderscores"
```

Comment:
We're actually generating client _and_ server here even though we only set `Server=true` (true is the default for both client and server).

I _think_ we only need to generate the server for `worker_service.proto` but need both for `benchmark_service.proto`.

Reply:
(Sorry, I didn't pick up on this in the last review!)